main.go

```diff
@@ -3,11 +3,12 @@ package main
 import (
 	"log"
 
-	"github.com/Azure/git2grid/actions"
```

Comment:
For better or worse, `buffalo` has prescribed a layout, and that includes keeping this folder named `actions` instead of `server`. Please revert that reorganization :)